pkgs/applications/networking/python-eduvpn-client/default.nix

```diff
@@ -0,0 +1,46 @@
+{ pkgs, stdenv, fetchFromGitHub, pythonPackages, gtk3, libnotify
+, gobjectIntrospection, networkmanager_openvpn, wrapGAppsHook
+, gdk_pixbuf }:
+
+  with pythonPackages;
```

Comment:
Better use `pythonPackages.buildPythonApplication` and drop this line.

Reply:
pythonPackages is needed in other places too. You made me realise that I could remove those as well.